### ISSUE TITLE
Errors while building

### ISSUE THREAD

**Issue by zobad16** [vdemydiuk/mtapi#281]
I am trying to build the source code from the dev branch. When I try to build the solution, following the instructions I get the following errors:

```
Build started at 3:56 PM...
1>------ Build started: Project: MtService, Configuration: Debug x64 ------
2>------ Build started: Project: MtClient, Configuration: Debug Any CPU ------
Restored C:\Users\ShenzhouCapital\Desktop\MTAPI\MtClient\MtClient.csproj (in 1 ms).
Restored C:\Users\ShenzhouCapital\Desktop\MTAPI\MtApi5\MtApi5.csproj (in 11 ms).
Restored C:\Users\ShenzhouCapital\Desktop\MTAPI\MtApi5Installer\MtApi5Installer.wixproj (in 13 ms).
Restored C:\Users\ShenzhouCapital\Desktop\MTAPI\MtApi\MtApi.csproj (in 14 ms).
Restored C:\Users\ShenzhouCapital\Desktop\MTAPI\TestClients\MtApi5TestClient\MtApi5TestClient.csproj (in 16 ms).
Restored C:\Users\ShenzhouCapital\Desktop\MTAPI\MtApiInstaller\MtApiInstaller.wixproj (in 18 ms).
Restored C:\Users\ShenzhouCapital\Desktop\MTAPI\TestClients\TestApiClientUI\TestApiClientUI.csproj (in 19 ms).
1>pch.cpp
1>LogConfigurator.cpp
2>MtClient -> C:\Users\ShenzhouCapital\Desktop\MTAPI\build\products\Debug\net8.0\MtClient.dll
3>------ Build started: Project: MtApi5, Configuration: Debug Any CPU ------
3>MtApi5 -> C:\Users\ShenzhouCapital\Desktop\MTAPI\build\products\Debug\net8.0\MtApi5.dll
1>Logger.cpp
1>MtConnection.cpp
1>MtExpert.cpp
1>MtMessage.cpp
1>MtServer.cpp
1>MtService.cpp
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.h(22,59): error C2039: 'string': is not a member of 'std'
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.h(22,48): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.h(22,65): error C2143: syntax error: missing ',' before '&'
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.h(23,46): error C2039: 'string': is not a member of 'std'
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.h(23,35): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.h(23,52): error C2143: syntax error: missing ',' before '&'
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.h(26,10): error C2039: 'string': is not a member of 'std'
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.h(26,17): error C3646: 'GetCommandPayload': unknown override specifier
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.h(26,34): error C2059: syntax error: '('
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.h(26,46): error C2238: unexpected token(s) preceding ';'
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.h(28,30): error C2039: 'string': is not a member of 'std'
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.h(28,19): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.h(28,36): error C2143: syntax error: missing ',' before '&'
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.cpp(209,17): error C2511: 'void MtService::SendEvent(int,int,const std::string &)': overloaded member function not found in 'MtService'
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.cpp(214,17): error C2511: 'void MtService::SendResponse(int,const std::string &)': overloaded member function not found in 'MtService'
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.cpp(224,24): error C2039: 'GetCommandPayload': is not a member of 'MtService'
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.cpp(226,12): error C2065: 'impl_': undeclared identifier
1>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtService\MtService.cpp(229,17): error C2511: 'void MtService::LogError(const std::string &)': overloaded member function not found in 'MtService'
1>Generating Code...
1>Done building project "MtService.vcxproj" -- FAILED.
4>------ Build started: Project: MT5Connector, Configuration: Debug x64 ------
4>Stdafx.cpp
4>MT5Connector.cpp
4>C:\Users\ShenzhouCapital\Desktop\MTAPI\MT5Connector\Mt5Handler.h(22,21): warning C4312: 'type cast': conversion from 'int' to 'HWND' of greater size
4>LINK : fatal error LNK1104: cannot open file 'C:\Users\ShenzhouCapital\Desktop\MTAPI\build\products\Debug\x64\MtService.lib'
4>Done building project "MT5Connector.vcxproj" -- FAILED.
5>------ Build started: Project: MtApi5Installer, Configuration: Release x64 ------
5>C:\Users\ShenzhouCapital\Desktop\MTAPI\MtApi5Installer\Product.wxs(98): error WIX0103: Cannot find the File file '..\build\products\Release\x64\MT5Connector.dll'. The following paths were checked: ..\build\products\Release\x64\MT5Connector.dll, C:\Users\ShenzhouCapital\Desktop\MTAPI\build\products\Debug\x64\..\build\products\Release\x64\MT5Connector.dll, C:\Users\ShenzhouCapital\Desktop\MTAPI\build\products\Debug\net8.0\..\build\products\Release\x64\MT5Connector.dll
5>Done building project "MtApi5Installer.wixproj" -- FAILED.
========== Build: 2 succeeded, 3 failed, 0 up-to-date, 0 skipped ==========
========== Build completed at 3:56 PM and took 17.307 seconds ==========
```

Any help would be appreciated.

**Comment by vdemydiuk** [comment]
@zobad16 is this build error still available for you?

**Comment by vdemydiuk** [comment]
The build was fixed in b89f00d31028ba9325ee30cc6309fc964eac8a0a.
